src/dash-table/components/Table/index.tsx

```diff
@@ -103,15 +129,18 @@ export default class Table extends Component<PropsWithDefaultsAndDerived> {
         const visibleColumns = this.visibleColumns(columns);
 
         return R.mergeAll([
-            props,
+            this.props,
+            this.state,
             {
                 columns: visibleColumns,
                 paginator,
                 setProps,
+                setState,
                 viewport,
                 viewport_selected_rows,
                 virtual,
-                virtual_selected_rows
+                virtual_selected_rows,
+                virtualized
```

Comment:
The ControlledTable's props are now a combination of all props and state (for standalone) + all derived props

Reply:
Note that `virtualized_data` is *not* part of the data contract with Dash -- this is purely internal